config.json

```diff
@@ -42,8 +42,8 @@
       "values": ["True", "False"],
       "defaultValue": true
     },
-    "TestArchitecture": {
-      "description": "Sets the architecture value that will be used for testing.",
+    "ArchitectureGroup": {
```

Comment:
What @chcosta said. I plan to take a closer look at the scenarios and come up with a recommendation but this should be OK for now.